examples/2d/dynamic_text2d.rs

```diff
@@ -0,0 +1,200 @@
+use bevy::prelude::*;
+use bevy_internal::sprite::Anchor;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_systems(Startup, setup)
+        .add_systems(Update, move_anchor_point)
+        .add_systems(Update, update)
+        .add_systems(Update, handle_button)
+        .run();
+}
+
+#[derive(Component)]
+struct DynamicText;
+
+#[derive(Resource)]
+struct TextStyleRes {
+    style: TextStyle,
+}
+
+#[derive(Component)]
+struct AnchorMarker;
+
+#[derive(Component)]
+struct AnchorDescriptor;
+
+fn update(
+    mut gizmos: Gizmos,
+    anchor_pos_query: Query<&Transform, With<DynamicText>>,
+    mut anchor_descriptor_query: Query<
+        &mut Transform,
+        (With<AnchorDescriptor>, Without<DynamicText>), // (Without<DynamicText>, Without<AnchorMarker>, With<Text>),
+    >,
+) {
+    if let Ok(&transform) = anchor_pos_query.get_single() {
+        gizmos.ray_2d(
+            Vec2::new(transform.translation.x, transform.translation.y),
+            Vec2::new(100.0, 245.0),
+            Color::RED,
+        );
+        for mut desc_transform in &mut anchor_descriptor_query {
+            *desc_transform = Transform::from_xyz(
+                transform.translation.x + 100.0,
+                transform.translation.y + 290.0,
+                0.0,
+            );
+        }
+    }
+}
+
+fn move_anchor_point(
+    mut anchor_query: Query<&mut Transform, With<DynamicText>>,
+    keyboard_input: Res<Input<KeyCode>>,
+    time: Res<Time>,
+) {
+    let mut move_dir = Vec3::ZERO;
+    if keyboard_input.pressed(KeyCode::W) {
```

Comment:
Can you use arrow keys instead? They'll play nicer on non-English keyboard layouts.